src/components/form.module.css

```diff
@@ -1,5 +1,13 @@
+.container {
+  margin: 2.313rem 2.25rem 0;
+}
+
+.formContainer {
+  width: 95vw;
+}
+
```

Comment:
- [ ] Consider editing this part near the form so that it looks like the Zeplin design.

- your design
![image](https://user-images.githubusercontent.com/52143815/193367207-3a07439f-3b94-41da-858d-73b1a4f8f6a1.png)

- Figma design
![image](https://user-images.githubusercontent.com/52143815/193367264-5d4722d0-6df2-458e-93b4-16a2c446bed1.png)